### PR TITLE
fix: errors on profil management form from toast to inline, restyle s…

### PR DESCRIPTION
**Pull request by cociugsergiu**
…uccess toast, replace email placeholder in change email confirmation flow

### CODE REVIEW

File: src/pages/ProfileManagement/components/ChangePasswordModalForm.js [hunk 1]
```diff
@@ -73,11 +72,18 @@ const ChangePasswordModalForm = ({toggleModal}) => {
         history.push(FORGOT_PASSWORD_PATH)
       }
       if (response.error) {
-        showErrorToast(<FormattedMessage id={response.error} />)
+        if (response.error === 'pleaseEnterCurrentPassword') {
```
Comment (MihailGorceagEndava) [inline src/pages/ProfileManagement/components/ChangePasswordModalForm.js:75]: you dont need these two ifs, you can just do setError(response)

File: src/routing/index.js [hunk 2]
```diff
@@ -17,34 +17,54 @@ const pagesWhereBackgroundisOn = [LOGIN_PATH, FORGOT_PASSWORD_PATH, CHANGE_PASSW
...
+          <div className='login-form-container'>
+            <div style={{ width: '100%' }}>
+              {isAuthenticated ?
+                <AuthorizedApp />
+                : <AnonymousApp />}
+            </div>
+          </div>
+          <div className='page-background-bird-login-page'></div>
+        </div>) :
+        <div className={pagesWhereBackgroundisOn.includes(location.pathname) ? 'page-background-bird' : 'page-background'}>
```
Comment (MihailGorceagEndava) [inline src/routing/index.js:58]: only here logic should be rewriten, in this className

File: src/pages/ProfileManagement/components/ChangePasswordModalForm.js (hunk 1 above)
Comment (cociugsergiu) [reply src/pages/ProfileManagement/components/ChangePasswordModalForm.js]: fixed

File: src/pages/ProfileManagement/components/ChangePasswordModalForm.js [hunk 3]
```diff
@@ -73,11 +72,14 @@ const ChangePasswordModalForm = ({toggleModal}) => {
         history.push(FORGOT_PASSWORD_PATH)
       }
       if (response.error) {
-        showErrorToast(<FormattedMessage id={response.error} />)
+        setError({ field: 'password', error: 'enterOldPassword' })
```
Comment (MihailGorceagEndava) [inline src/pages/ProfileManagement/components/ChangePasswordModalForm.js:75]: you can just write setError(response)

Comment (cociugsergiu) [reply src/pages/ProfileManagement/components/ChangePasswordModalForm.js:75]: fixed

**Pull request merged**